shell/completion.bash

```diff
@@ -278,7 +278,7 @@ a_cmds="
   basename bunzip2 bzip2 chmod chown curl cp dirname du
   find git grep gunzip gzip hg jar
   ln ls mv open rm rsync scp
-  svn tar unzip zip"
+  svn tar tig unzip zip"
```

Comment:
@junegunn Perhaps `a_cmd` could include an environment variable, e.g. `a_cmds=${FZF_COMPLETION_COMMANDS} awk cat ...`, so that it's easy extend any command with fzf completion?

My use case is I'd like fzf completion on `**<TAB>` for a custom defined alias that has existing tab completion.

Reply:
@ptzz Yeah, that seems like a better idea.

Reply:
@ptzz But then again, I'm not sure if we really need another variable when it's already possible to set up fuzzy completion for individual commands.

https://github.com/junegunn/fzf#supported-commands

Reply:
@junegunn IIRC original tab completions were not preserved when setting up fuzzy completion for new commands.

Reply:
Good point. I think a better approach would be to provide an API for setting up completion, but that needs more discussion.

Reply:
For anyone interested in a workaround/hack not requiring changes to `fzf`, I currently have this in my `.bash_profile`:

```
_fzf_orig_completion_tig=_tig
complete -F _fzf_path_completion -o default -o bashdefault tig
```